src/components/SharedComponents/PhotoCount.js

```diff
@@ -3,23 +3,29 @@
 import { View } from "components/styledComponents";
 import * as React from "react";
 import Svg, { Path, Text } from "react-native-svg";
+import { viewStyles } from "styles/sharedComponents/photoCount";
+import colors from "styles/tailwindColors";
```

Comment:
We are in the process of unifiying our color usage through the app. In the meantime could you please switch to import colors using the `useTheme` hook from react-native-paper. E.g. here: https://github.com/YahiaJabeur/iNaturalistReactNative/blob/33633c06f110878c0c9b117ea21f4fa6428bb9fb/src/components/UiLibrary.js#L47